packages/node_modules/nav-frontend-tabs/src/tabs-pure.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+import { guid, omit, keyCodes } from 'nav-frontend-js-utils';
+
+import Tab, {TabProps} from './tab';
+
+const tabsCls = (props) => cn('nav-frontend-tabs', props.className, {
+    'nav-frontend-tabs--kompakt': props.kompakt
+});
+
+export interface TabsPureProps {
+    tabs: TabProps[];
+    kompakt?: boolean;
+    manuell?: boolean;
+    onChange?: (index: number) => void;
+}
+
+class TabsPure extends React.PureComponent<TabsPureProps> {
+
+    private aktivIndex: number = 0;
+    private focusIndex: number = 0;
+    public linkRefs: any[] = [];
+
+    static Tab = Tab;
```

Comment:
Hvorfor trenger du en statisk variabel av denne?

Reply:
For å etterkomme ønsket om følgende API:
```JSX
import Tabs from 'nav-frontend-tabs';

<Tabs onChange={this.handleChange}>
    <Tabs.Tab>Min tab</Tabs.Tab>
</Tabs>
```
Altså dot-notasjon på `Tabs.Tab`.